scripts/archive.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+TOOL_NAME=XcodeGen
```

Comment:
Can we move this script into the make file. Many of these variables are repeated

Reply:
Added `export` in Makefile to share variables. d07f160
I have to fix code format for Makefile, so I keep the logic in separated script for now.